app/loginmodule/client.go

```diff
@@ -133,3 +183,31 @@ func recoverPanics(returnErr *error) { // nolint:gocritic
 		}
 	}
 }
+
+func decryptAes128Ecb(data, key []byte) []byte {
```

Comment:
I think it should be moved to a different file.

Reply:
fixed